app/controllers/organizations_controller.rb

```diff
@@ -5,6 +5,11 @@ class OrganizationsController < ApplicationController
   def index
   end
 
+  def new
+    raise PermissionError unless current_user.can_create_organization?
```

Comment:
:goberserk: Do we have to raise and blow up? Couldn't we just add a flash message and bounce the user back to where they came from?